srcpkgs/tlf/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'tlf'
+pkgname=tlf
+version=1.4.0
+revision=1
+build_style=gnu-configure
+configure_args="--datadir=/usr/share --enable-fldigi-xmlrpc"
+make_install_args="prefix=${DESTDIR}/usr datadir=${DESTDIR}/usr/share"
```

Comment:
I also doubt this is necessary

Reply:
I was following the upstream build instructions

Reply:
Not needed, `build_style` will set these variables.